### PR TITLE
Typo

### PR DESCRIPTION
**Pull request by BillyZac**

### CODE REVIEW

File: examples/profile-cards-primitives/README.md [hunk 1]
```diff
@@ -36,6 +36,6 @@ Open a browser to `http://localhost:3000`
 
 ## The idea behind the example
 
-`react-primitives` provides a powerful way to build platform-independent components. This is a simple example of rendering to web & Sketch simulatenously.
+`react-primitives` provides a powerful way to build platform-independent components. This is a simple example of rendering to web & Sketch simultaneously.
 
-![examples-primitives](https://cloud.githubusercontent.com/assets/591643/24778174/0dd9d214-1ade-11e7-9372-fa8b578a0b48.png)
\ No newline at end of file
+![examples-primitives](https://cloud.githubusercontent.com/assets/591643/24778174/0dd9d214-1ade-11e7-9372-fa8b578a0b48.png)
```
Comment (tylerhowarth) [inline examples/profile-cards-primitives/README.md]: Did something change on this line or with the image?

Comment (jongold) [inline examples/profile-cards-primitives/README.md]: Maybe whitespace? :shipit:

**Comment by repus** [comment]
erightsoft.com©>>♥
-------- Original message --------From: Jon Gold <notifications@github.com> Date: 4/27/17  12:17 AM  (GMT-06:00) To: airbnb/react-sketchapp <react-sketchapp@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: Re: [airbnb/react-sketchapp] Typo (#35) 
Merged #35.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/airbnb/react-sketchapp","title":"airbnb/react-sketchapp","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/airbnb/react-sketchapp"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"Merged #35."}],"action":{"name":"View Pull Request","url":"https://github.com/airbnb/react-sketchapp/pull/35#event-1059788534"}}}

File: examples/profile-cards-primitives/README.md (hunk 1 above)
Comment (repus) [inline examples/profile-cards-primitives/README.md]: Who /*\

**Pull request merged**